core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockWorkspaceManagerDAO.scala

```diff
@@ -100,6 +100,10 @@ class MockWorkspaceManagerDAO(val createCloudContextResult: CreateCloudContextRe
   override def getWorkspaceCreateCloudContextResult(workspaceId: UUID,
                                                     jobControlId: String,
                                                     accessToken: OAuth2BearerToken): CreateCloudContextResult = mockCreateAzureCloudContextResult()
+
+  override def enableApplication(workspaceId: UUID, applicationId: String, accessToken: OAuth2BearerToken): WorkspaceApplicationDescription = {
+    new WorkspaceApplicationDescription().workspaceId(workspaceId).applicationId(applicationId)
```

Comment:
I don't understand what this is doing. Can you explain?

Reply:
This is returning an instance of a workspace application model as would be instantiated by the real http workspace manager dao. In this case, it's simply giving back the expected workspaceId and applicationId.

Reply:
Another question-- how does the chaining work here? And does it mean that chaining could be done up above where I complained about the lack of an explicit `return`?